src/Sylius/Bundle/ProductBundle/EventListener/ElasticaProductListener.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ProductBundle\EventListener;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Doctrine\ORM\Event\PostFlushEventArgs;
+use FOS\ElasticaBundle\Persister\ObjectPersister;
+use Sylius\Component\Product\Model\AttributeValueInterface;
+use Sylius\Component\Core\Model\ProductTranslation;
+use Sylius\Component\Variation\Model\VariantInterface;
+
+/**
+ * Elastica product listener
+ *
+ * @author Nicolas Adler <nicolas.adler@openizi.com>
+ */
+class ElasticaProductListener implements EventSubscriber
+{
+    /**
+     * @var ObjectPersister
+     */
+    protected $objectPersister;
+
+    /**
+     * Objects scheduled for update
```

Comment:
It would be better to place a `var` statement here rather than redundant comment